openpype/hosts/blender/api/pipeline.py

```diff
@@ -209,11 +210,10 @@ def reload_pipeline(*args):
 
     """
 
-    avalon.api.uninstall()
+    uninstall_host()
```

Comment:
Even though this is refactored now I don't believe this `reload_pipeline` still makes sense? Does it even trigger an install?

Should we just be removing "Reload pipeline" logic? Not sure if it's ever used and whether it even works. :D

Reply:
I even don't understand why `uninstall` is needed :D so from my point of view, yes.

Reply:
👍 +1 from me to kill the Reload Pipeline code with fire. 🔥